cluster/senza-definition.yaml

```diff
@@ -114,6 +114,10 @@ SenzaComponents:
           Value: owned
         - Key: "NodePool"
           Value: "{{ Arguments.WorkerNodePoolName }}"
+        - Key: "k8s.io/cluster-autoscaler/enabled"
+          Value: ""
+        - Key: "zalando.de/cluster-local-id/{{SenzaInfo.StackName}}-{{SenzaInfo.StackVersion}}"
```

Comment:
If the parsing is fixed in the autoscaler we could get rid of this right?

Reply:
Yes, it's not used by anything else.